ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/remove_container/scripts/deleteNodes.py

```diff
@@ -0,0 +1,59 @@
+import jenkins
+import sys
+import json
+import requests
+
+def createServer(username, password):
+    url = "http://ci.adoptium.net:80"
```

Comment:
This should be using https I'd have thought - we can't have credentials being passed across unencrypted. We're using https for the `doDelete` operation later on in this file.

Reply:
Good spot